spec/controllers/api/v1/users/checkins_controller_spec.rb

```diff
@@ -66,32 +67,32 @@
 
       it 'fetches the last reported location (public attributes only)' do
         get :last, params: params
-        expect(res_hash.first['lat']).to be checkin.lat
-        expect(res_hash.first.keys).not_to include(*private_checkin_attributes)
+        expect(res_hash[:checkins].first['lat']).to be checkin.lat
```

Comment:
Format of response changes since we introduced AM::Serializers.
It now has root element that is "checkins" for checkins controller.
It can be managed by adapter [here](https://github.com/earlymarket/Coposition/pull/664/files#diff-97e67623d81b040290a5bc14a3711323R2).